server/routes/auth.js

```diff
@@ -41,6 +42,7 @@ router.post('/api/user/create', authMiddleware, async (req, res) => {
         })
         console.log("saving user")
         await user.save();
+        res.cookie('didToken',didToken,{httpOnly:true})
```

Comment:
Explain this

Reply:
Remember I commented the `window.localStorage.setItem("didToken", newDidToken);`  well I did that for this. This is setting the token ass httpOnly cookie in the response header so whenever client receives this response , client will automatically save the token in the browser cookie and I have passed `httpOnly:true` because by mentioning this it will ensure that it will not be accessible from client javascript.